clusterloader/framework/rc_utils.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"fmt"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/kubernetes/pkg/api/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+	kutils "k8s.io/kubernetes/test/utils"
+)
+
+// CreateRC will create a new RC if it does not exist, or it will update an existing RC with a new replica count if it does exist
+func CreateRC(f *framework.Framework, namespace, name, imageName string, label map[string]string, replicas int) error {
+	rc, _ := f.ClientSet.Core().ReplicationControllers(namespace).Get(name, metav1.GetOptions{})
+	if rc.Name == "" {
```

Comment:
This isn't correct - this would never be the case.
Basically Get() will either return a correct RC (which has to have name) or an error "notexisting" (you can look for it in main repo probably).